service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/JudicialMessageContentProviderTest.java

```diff
@@ -1,25 +1,41 @@
 package uk.gov.hmcts.reform.fpl.service.email.content;
 
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
 import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.HearingBooking;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
 import uk.gov.hmcts.reform.fpl.model.judicialmessage.JudicialMessage;
 import uk.gov.hmcts.reform.fpl.model.notify.NewJudicialMessageTemplate;
+import uk.gov.hmcts.reform.fpl.service.time.Time;
+import uk.gov.hmcts.reform.fpl.utils.FixedTimeConfiguration;
+
+import java.time.LocalDateTime;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static uk.gov.hmcts.reform.fpl.enums.YesNo.NO;
 import static uk.gov.hmcts.reform.fpl.enums.YesNo.YES;
-import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.populatedCaseData;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
 
-@ContextConfiguration(classes = {JudicialMessageContentProvider.class})
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {JudicialMessageContentProvider.class, FixedTimeConfiguration.class})
 class JudicialMessageContentProviderTest extends AbstractEmailContentProviderTest {
     @Autowired
     private JudicialMessageContentProvider newJudicialMessageContentProvider;
 
+    @Autowired
+    Time time;
+
+    //Tech debt - maybe mock callout instead of setting date to arbitrary far date in future
```

Comment:
you can use mockstatic with GeneratedOrderTest or you can transform your helper into an object and remove the static. Otherwise create a jira ticket and remove the comment

Reply:
turns out didnt need mockstatic in the end, but learnt how to do it 👍